### ISSUE TITLE
Error for generating compute_detection_metrics_main

### ISSUE THREAD

**Issue by coralemon** [open-mmlab/mmdetection3d#1187]
when I use
```
bazel build waymo_open_dataset/metrics/tools/compute_detection_metrics_main
```
to generate compute_detection_metrics_main file I receive following error and need for help
```
+ bazel build waymo_open_dataset/metrics/tools/compute_detection_metrics_main
INFO: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /root/.cache/bazel/_bazel_root/61cc9052d346f6d8b31c2c1f1f71a52e/external/bazel_tools/tools/build_defs/repo/http.bzl:237:16):
 - /root/mmdetection-lib/waymo-od/tf/workspace_tf2_6_0.bzl:39:5
 - /root/mmdetection-lib/waymo-od/WORKSPACE:72:1
ERROR: /root/mmdetection-lib/waymo-od/waymo_open_dataset/metrics/tools/BUILD:17:1: error loading package '@com_google_absl//absl/strings': Unable to find package for @rules_cc//cc:defs.bzl: The repository '@rules_cc' could not be resolved. and referenced by '//waymo_open_dataset/metrics/tools:compute_detection_metrics_main'
ERROR: Analysis of target '//waymo_open_dataset/metrics/tools:compute_detection_metrics_main' failed; build aborted: error loading package '@com_google_absl//absl/strings': Unable to find package for @rules_cc//cc:defs.bzl: The repository '@rules_cc' could not be resolved.
INFO: Elapsed time: 0.160s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded, 0 targets configured)
    currently loading: waymo_open_dataset ... (3 packages)
```

**Comment by ZCMax** [comment]
Maybe you can refer to: https://github.com/waymo-research/waymo-open-dataset/issues/11 for solution. Feel free to leave comments if it doesn't wok.

**Comment by coralemon** [comment]
Thanks! I found it may loss some dependency and I add 
```
http_archive(
    name = "rules_proto",
    sha256 = "66bfdf8782796239d3875d37e7de19b1d94301e8972b3cbd2446b332429b4df1",
    strip_prefix = "rules_proto-4.0.0",
    urls = [
        "https://mirror.bazel.build/github.com/bazelbuild/rules_proto/archive/refs/tags/4.0.0.tar.gz",
        "https://github.com/bazelbuild/rules_proto/archive/refs/tags/4.0.0.tar.gz",
    ],
)

http_archive(
    name = "rules_java",
    sha256 = "ccf00372878d141f7d5568cedc4c42ad4811ba367ea3e26bc7c43445bbc52895",
    strip_prefix = "rules_java-d7bf804c8731edd232cb061cb2a9fe003a85d8ee",
    urls = [
        "https://mirror.bazel.build/github.com/bazelbuild/rules_java/archive/d7bf804c8731edd232cb061cb2a9fe003a85d8ee.tar.gz",
        "https://github.com/bazelbuild/rules_java/archive/d7bf804c8731edd232cb061cb2a9fe003a85d8ee.tar.gz",
    ],
)

http_archive(
        name = "rules_cc",
        urls = ["https://github.com/bazelbuild/rules_cc/archive/262ebec3c2296296526740db4aefce68c80de7fa.tar.gz"],
        strip_prefix = "rules_cc-262ebec3c2296296526740db4aefce68c80de7fa",
        sha256 = "3057c13fa4d431eb0e7a9c28eea13f25987d29f869406b5ee3f2bd9c4134cb0c",
    )
``` 
in *WORKSPACE* and this  problem has been solved.
And I think the reason may be the incorrect version of Bazel 
```wget https://github.com/bazelbuild/bazel/releases/download/0.28.0/bazel-0.28.0-installer-linux-x86_64.sh``` in mmdet3d doc and ```BAZEL_VERSION=3.1.0``` in Waymo-od doc ( but I didn't try for new version it but manually add serval dependency in version 2.8.0）

And I hope the instruction on mmdetection3d can be updated on this part ~ 
Thanks very much !!! ~

**Comment by Tai-Wang** [comment]
Thanks for your report. It might be caused by some updates from Waymo (When I developed this part before, I did not encounter such problems). We will reproduce your problems and update the documentation accordingly.

**Comment by ZCMax** [comment]
I have checked the problem, it's indeed caused by the incorrect version of Bazel. Updating bazel version to 3.1.0 can fix the problem, it would be fixed in #1223 .

**Comment by Tai-Wang** [comment]
Fixed in #1223 .
